src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -36,19 +37,42 @@ const PathwaysList: FC<PathwaysListProps> = ({
   service,
   resources
 }) => {
+  const [criteria, setCriteria] = useState<CriteriaResult[] | null>(null);
+
+  if (!criteria && evaluatedPathways.length > 0 && resources != null && resources.length > 0) {
```

Comment:
```suggestion
  if (!criteria && evaluatedPathways.length > 0 && resources && resources.length > 0) {
```
don't need to check against null, it will be considered true if it's not falsy.